crates/ink/Cargo.toml

```diff
@@ -1,38 +1,40 @@
 [package]
 name = "ink"
-version = "4.2.0"
-authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
-edition = "2021"
-rust-version = "1.63"
 
-license = "Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/paritytech/ink"
-documentation = "https://docs.rs/ink"
-homepage = "https://www.parity.io/"
+edition.workspace = true
+categories.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+version.workspace = true
+
+authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
 description = "[ink!] Rust based eDSL for writing smart contracts for Substrate"
-keywords = ["wasm", "parity", "webassembly", "blockchain", "edsl"]
-categories = ["no-std", "embedded"]
+documentation = "https://docs.rs/ink"
 include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
+readme = "README.md"
+rust-version = "1.63"
 
 [dependencies]
-ink_env = { version = "4.2.0", path = "../env", default-features = false }
-ink_storage = { version = "4.2.0", path = "../storage", default-features = false }
-ink_primitives = { version = "4.2.0", path = "../primitives", default-features = false }
-ink_metadata = { version = "4.2.0", path = "../metadata", default-features = false, optional = true }
-ink_prelude = { version = "4.2.0", path = "../prelude", default-features = false }
-ink_macro = { version = "4.2.0", path = "macro", default-features = false }
-
+derive_more = { workspace = true, features = ["from"] }
```

Comment:
Could you keep the order the same for now, please? Let's change it in a separate PR. We can use `cargo-sort` and include it in our CI